packages/admin/cms-admin/src/generator/future/generateGrid.ts

```diff
@@ -699,11 +699,9 @@ export function generateGrid(
                                     </>
                                 )`
                             : undefined,
-                        headerName: !column.headerInfoTooltip
-                            ? `intl.formatMessage({ id: "${instanceGqlType}.${column.name}", defaultMessage: "${
-                                  column.headerName || camelCaseToHumanReadable(column.name)
-                              }" })`
-                            : undefined,
+                        headerName: `intl.formatMessage({ id: "${instanceGqlType}.${column.name}", defaultMessage: "${
```

Comment:
we now have the formatMessage twice. Could we use headerName inside reanderHeader? (`params.colDef.headerName`) that would avoid duplicating the headerName